BWEnv/src/utils.cc

```diff
@@ -270,15 +268,15 @@ std::string readIni(const char* filename, const char* section, const char* key)
       while (c != e && *c != ']' && *c != '\n') {
         ++c;
       }
-      if (size_t(c - n) == section_strlen && !memcmp(n, section, section_strlen)) correct_section = true;
+      if (size_t(c - n) == section.size() && !memcmp(n, section.c_str(), section.size())) correct_section = true;
```

Comment:
If you prefer it like this, sure. If I could use c++17 I would use string_view :D